rust/kcl-lib/src/execution/exec_ast.rs

```diff
@@ -2969,6 +2975,22 @@ impl Node<PipeExpression> {
     }
 }
 
+pub(crate) struct Solved {
```

Comment:
Even though we use it here, maybe it would make more sense to move this into `rust/kcl-lib/src/execution/sketch_solve.rs`. Then add a `From<kcl_ezpz::SolveOutcome>` impl.